image/tarexport/save.go

```diff
@@ -288,10 +288,11 @@ func (s *saveSession) saveImage(id image.ID) (map[layer.DiffID]distribution.Desc
 	var layers []string
 	var foreignSrcs map[layer.DiffID]distribution.Descriptor
 	for i := range img.RootFS.DiffIDs {
+		v1ImgCreated := time.Unix(0, 0)
```

Comment:
I guess this one can be created outside of the loop 😅

Reply:
Doh, just went for the closest scope because I couldn't `&time.Unix(0, 0)` and didn't even think about it being a loop.  Is it a problem for each iteration to point to the _same_ instance of `time.Unix(0, 0)`?

Reply:
It should _probably_ be fine to re-use? But creating a new `time.Unix()` in the global scope of an `docker export` is probably the least of our concerns on performance 😂

Reply:
Arguably, we could use `nil` here instead, but it's got a higher chance of breaking something than an explicit `0` value.